### ISSUE TITLE
Exclude raw markdown and table-headers in search result

### ISSUE THREAD

**Issue by renegadevi** [leo-buneev/vuepress-plugin-fulltext-search#9]
When searching tables, it includes Table headers and raw markdown syntax in the result. See below.

## This is example table I have in markdown
<img width="807" alt="table-example" src="https://user-images.githubusercontent.com/10997469/82159712-98883f80-9890-11ea-9c70-d5eb5bbf6de0.png">

### Table source code
```md
## Backup / Cloning
| Name ⇵ | Description ⇵ | Get it ⇵ | ⇵ | ⇵ |
|:-------|:--------------|:--------:|:-:|:-:|
|[Carbon Copy Cloner](https://bombich.com/)| Superb bootable backup solution for the Mac|[Website](https://bombich.com/)|![Paid](symbols/paid.svg "Paid")|![Closed-Source](symbols/closed.svg "Closed-Source")|
|[SuperDuper](https://www.shirt-pocket.com/SuperDuper/SuperDuperDescription.html)| Advanced, yet easy to use disk copying program for macOS.|[Website](https://www.shirt-pocket.com/SuperDuper/SuperDuperDescription.html)|![Paid](symbols/paid.svg "Paid")|![Closed-Source](symbols/closed.svg "Closed-Source")|
```

## Expected result (Photoshopped)
First with either just first table data with has the word, or as the second result, the row of the result. 
<img width="534" alt="carbon-search-expected-result" src="https://user-images.githubusercontent.com/10997469/82159914-084afa00-9892-11ea-9f05-76b811e811c0.png">

## Actual result

<img width="534" alt="carbon-search" src="https://user-images.githubusercontent.com/10997469/82159736-cff6ec00-9890-11ea-812a-6b718d1c1ea4.png">

**Comment by czif** [comment]
Yeah

**Comment by leo-buneev** [comment]
Resolved in 2.0.4. Results won't contain raw markdown anymore. However, decided to leave table headers - could be. useful to index them, and they don't look as bad in search results anymore.
